context/app/static/js/components/files/FileBrowserDirectory/FileBrowserDirectory.jsx

```diff
@@ -13,20 +16,35 @@ function FileBrowserDirectory(props) {
     }
   };
 
+  const displayOnlyQaQc = useFilesStore(filesStoreSelector);
+
+  useEffect(() => {
+    if (displayOnlyQaQc === true) {
+      setIsExpanded(true);
+      return;
+    }
+    setIsExpanded(false);
```

Comment:
or...
```suggestion
    setIsExpanded(displayOnlyQaQc);
```
or...
```suggestion
    setIsExpanded(Boolean(displayOnlyQaQc));
```

If there's some other case you're considering that would break these, please add a comment to explain.